frontend/packages/console-shared/src/components/dashboard/inventory-card/InventoryItem.tsx

```diff
@@ -193,6 +193,7 @@ const ResourceTitleComponent: React.FC<ResourceTitleComponentComponent> = ({
 export const ResourceInventoryItem: React.FC<ResourceInventoryItemProps> = ({
   kind,
   useAbbr,
+  titleComponent,
```

Comment:
```suggestion
  TitleComponent,
```
since its a Component